components/dashboard/filters/Filterbar.tsx

```diff
@@ -117,7 +117,7 @@ export function Filterbar<FV extends Record<string, any>, FM>({
           )}
         </div>
         {filters.orderBy && (
-          <div className="flex flex-1 justify-end">
+          <div className="flex flex-1">
```

Comment:
While we might want to improve how the filters wrap and break together with the order filter, I still think we want to keep the order filter aligned to the right, to give it a consistent expected position across all pages and more importantly have the concept of **sorting** slightly set apart from the concept of **filtering**.

One thing to fix here though, to allow truncating long Order filter labels on small screens we can set "w-full" here:

```suggestion
          <div className="flex w-full flex-1 justify-end">
```

Reply:
I just tested this change (`w-full`) and it makes the sorting filter always break to the next line. Is that expected?

![image](https://github.com/opencollective/opencollective-frontend/assets/50931490/2080a819-26c8-428c-bf33-2bfff8bf59eb)

Reply:
Actually, nevermind. The css was wrong when testing this.